### ISSUE TITLE
Add Firefox mobile Integration

### ISSUE THREAD

**Issue by Y0ngg4n** [ingenitdo/Nervenschoner#1]
It would be nice to have this plugin on android and ios with the Firefox Mobile App too.

**Comment by carlolf** [comment]
yes please, make this available for Firefox Android / Firefox Klar.

**Comment by c-hartmann** [comment]
that was my first thinking as well. uBlock *does* it in Firefox Mobile. so Nervenschoner (best name ever) should do it too
